day03/ex00/ex00.md

```diff
@@ -0,0 +1,143 @@
+
+![billing section](../assets/billing.png){width=450px}
+
+Then we can go in `Billing preferences` section and validate `Receive Free Tier Usage Alerts` (we have to enter an email). We can also set `Receive Billing Alerts` then we have to save preferences. We can now receive billing alerts !
+
+Now we need to go CloudWatch to configure a billing alert. Cloudwatch is the AWS monitoring system use to track performances of used AWS resources.
+
+We go into `CloudWatch` section. We go into the `Alarm/Billing` subsection. Then we can `create alarm`.
+
+![alarm](../assets/account_alarm.png){width=450px}
+
+We have to `select metric`. Two types of metric exists : `by service` or `Total estimated charges`. We are going to select `Total estimated charges` and `USD` as currency. The threshold we want is `static` and `greater or equal` to 1 $.
+
+![alarm condition](../assets/account_alarm_cond.png){width=450px}
+
+On the configure actions page. we are going to keep `in alarm`. For the notification we use SNS (Simple Notification Service). We are going to `create a new topic` with the name "BillingAlarm" and we need to enter to enter our email address. Then we can create the SNS topic.
+
+For the SNS topic to work, we need to confirm our email address (confirmation in the AWS email).
+
+Then we can select `select an existing SNS topic` with "BillingAlarm".
+
+We can then add the name and the description of the alarm. Our alarm is finally created!
+
+## Create an IAM user
+
+AWS recommands we don't use our root account for day to day administrative tasks. For that, we are going to create a new IAM user and attach administrative access policy to it. as best practice AWS recommends we give users the minimum access to resources for users to accomplish their day to day tasks.
+
+First, go to the `IAM` service (IAM stands for Identity and access management). To create a user we go in the dashboard into the `users` section, then `add user`.
+
+![add user](../assets/add_user1.png){width=450px}
+
+We can enter the name of the new user. We have to select an access type. We select both `Programmatic access` and `AWS Management Console access`. Accesses will be seen later.
+We can then select an automatically generated password or custom password (here a custom password is used) we can force the change of the password when the user logs in.
+
+The second step is to set permissions for the new user. For the first user we are going to attach an `AdministratorAccess` policy (which gives the user a full access to all AWS services).
+
+Then a tag can be added to the user (optionnal). The last step reviews all the options set for the user. The user is finally created !
+
+\newpage
+
+![user review](../assets/add_user2.png){width=400px}
+
+At the end you will see the following screen. The access key is really important as it will be used during in the next exercise. Save the csv file otherwise you will nerver ever have access to the secret key again.
```

Comment:
```suggestion
At the end, you will see the following screen. The access key is really important as it will be used in the next exercise. Save the CSV file otherwise, you will never ever have access to the secret key again.
```